ledger/handle/get_children_test.go

```diff
@@ -0,0 +1,166 @@
+package handle
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+
+	"github.com/insolar/insolar/insolar/record"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/delegationtoken"
+	"github.com/insolar/insolar/insolar/flow/bus"
+	"github.com/insolar/insolar/insolar/jet"
+	"github.com/insolar/insolar/insolar/message"
+	"github.com/insolar/insolar/ledger/proc"
+	"github.com/insolar/insolar/ledger/recentstorage"
+	"github.com/insolar/insolar/ledger/storage"
+	"github.com/insolar/insolar/ledger/storage/object"
+	"github.com/insolar/insolar/testutils"
+)
+
```

Comment:
i thought that old test will still working and it's a sign, that nothing was broken

Reply:
Most of all tests still work. However unit tests were relaying on the method `handleGetChildren` that doesn't exists anymore thus I had to rewrite these unit tests.